jakartaee11/JakartaEE11ReleasePlan.md

```diff
@@ -198,61 +200,61 @@ As stated earlier, all component Specifications which plan a Major or Minor rele
 
 ### Waves
 
-We are proposing to deliver Jakarta EE 11 in a set of waves similar to those delivered in the previous Jakarta EE releases. These waves are somewhat related to the dependency tree of specifications. We aim to deliver specifications with a low number of dependencies first followed by other specifications.
+We are proposing to deliver Jakarta EE 11 in a set of waves similar to those delivered in the previous Jakarta EE releases. These waves are somewhat related to the dependency tree of specifications. We aim to deliver specifications with a low number of dependencies first followed by other specifications. (updated specifications marked with asterisks)
 
 #### Wave 1
 
-- Jakarta Annotations
+- Jakarta Annotations*
+- Jakarta JSON Processing* (service release)
 
 #### Wave 2
 
-- Jakarta Expression Language
-- Jakarta Interceptors
+- Jakarta Expression Language*
+- Jakarta Interceptors*
 - Jakarta Lang Model (may be released with Jakarta CDI [Core] in wave 3)
```

Comment:
```suggestion
- Jakarta Lang Model (may be released with Jakarta CDI [Core] in wave 3) *
```

Reply:
It has to be released as it is part of CDI.

Reply:
Fixed in fe99cc4.